srv/components/provisioner/config.sls

```diff
@@ -28,8 +28,13 @@ provisioner_rsyslog_conf_updated:
     - watch_in:
       - service: Start rsyslog
 
-# Always start glusterfshsaredstorage before salt-minion
-Update glusterfshsaredstorage.service:
+# Always start glusterfshsaredstorage before salt-master
```

Comment:
Sequence-wise should we write GlusterFS info before this Salt-Master section

Reply:
@nitin-seagate here were are just copying service files from template location to target location.
No service is being restarted.

However, I've sequenced the states appropriately.